.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Does this mean pull requests to or from master? We're rarely going to have a pull request from master.

Reply:
Should be to `master`